README.md

```diff
@@ -67,6 +67,29 @@ Notice that in this case it is important to use the `prefix` option to tell the
 
 For other examples, see `example.js`.
 
+## Request tracking
+
+`fastify-http-proxy` can track and pipe the `request-id` across the upstreams. Using the [`hyperid`](https://www.npmjs.com/package/hyperid) module and the [`fastify-reply-from`](https://github.com/fastify/fastify-reply-from) built in options a fairly simple example would look like this:
+
+```js
+const Fastify = require('fastify')
+const proxy = require('fastify-http-proxy')
+const hyperid = require('hyperid')
+
+const server = Fastify()
+const uuid = hyperid()
+
+server.register(proxy, {
+  upstream: 'http://localhost:4001',
+  replyOptions: {
+    rewriteRequestHeaders: (originalReq, headers) => ({...headers, 'request-id': uuid()})
```

Comment:
I think this might be a bit more complex than that. This example override the request-id, however we should 1) change the fastify id generator to use the unique id and 2) take it from `req.id` instead of generating a new one each time.

It would be really nice to have a unit test for this.

Reply:
The Fastify instance should be configured to read the request id from the incoming header. https://www.fastify.io/docs/v2.12.x/Server/#requestidheader